server/routes/search.js

```diff
@@ -33,7 +36,10 @@ router.post(['/search/results', '/api/search/results'],
                 correspondentName: formData['correspondent'],
                 topic: formData['topic'],
                 data: {
-                    POTeamUUID: formData['signOffMinister']
+                    POTeamUUID: formData['signOffMinister'],
```

Comment:
mainly existing code, but I think this handler should definitely be in a middleware with some tests

Reply:
resolved with searchHandler component